data-bulletproofing.md

```diff
@@ -69,6 +69,13 @@ _Russell Clemmings of the Fresno Bee on rechecking your data:_
 - Have someone who knows the data check your results before publication -- even the target of the story, if possible.
 - Double-check surprising results -- if citations spiked by 50 percent in one year, it could be a story or it could (more likely) be an error. 
 
+_Peter Harkins, formerly of the Washington Post on structuring data:_
+
+- Extract real data (choose the worst, noisiest stuff) to use as fixtures in automated tests and you'll know you haven't broken things as you go.
+- Don't model anything as many-to-many. Name the intermediate concept and give it two one-to-many associations. It will almost always want to accumulate more info in the future, and the cost of changing your model will be high.
```

Comment:
I'm not sure I fully understand this point. In what situation would a many-to-many relationship *not* have an intermediate table / linking vehicle like you describe?